setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name="lookml-generator",
-    python_requires=">=3.8.0",
+    python_requires=">=3.7.0",
```

Comment:
This is a little unfortunate, but netlify doesn't currently support any version of python greater than 3.7 currently. I hope it will soon.